_includes/topnav.html

```diff
@@ -1,106 +1,37 @@
+<!-- Mobile Side Navigation -->
+<nav class="mobile_expanded" role="navigation">
+    <a href="{{ "/frequently-asked-questions.html" | prepend:site.baseurl }}" class="menu-item">FAQ</a>
+    <a href="http://www.cockroachlabs.com/docs/" class="menu-item">Docs</a>
+    <a href="http://www.cockroachlabs.com/community/" class="menu-item">Community</a>
+    <a href="http://www.cockroachlabs.com/about/" class="menu-item">About</a>
+    <a href="http://www.cockroachlabs.com/blog/" class="menu-item">Blog</a>
+    <a href="{{ "/install-cockroachdb.html" | prepend:site.baseurl }}" class="menu-item">Install</a>
+</nav>
+
+<!-- Desktop / Tablet Navigation -->
+<header class="navbar-fixed-top">
+    <div class="container">
+        <div class="row">
+            <a class="logo" href="/" title="Cockroach Labs">
+                <img class="default" src="http://www.cockroachlabs.com/wp-content/themes/cockroach-labs/images/logo.png" width="100%" alt="Cockroach Labs" />
+                <img class="scrolled" src="http://www.cockroachlabs.com/wp-content/themes/cockroach-labs/images/CL_Logo_Horizontal.png" width="100%" alt="Cockroach Labs" />
+            </a>
+
+            <nav class="desktop" role="navigation">
+                <a href="{{ "/frequently-asked-questions.html" | prepend:site.baseurl }}" class="menu-item">FAQ</a>
+                <a href="http://www.cockroachlabs.com/docs/" class="menu-item current-menu-item">Docs</a>
+                <a href="http://www.cockroachlabs.com/community/" class="menu-item">Community</a>
+                <a href="http://www.cockroachlabs.com/about/" class="menu-item">About</a>
+                <a href="http://www.cockroachlabs.com/blog/" class="menu-item">Blog</a>
+                <a href="{{ "/install-cockroachdb.html" | prepend:site.baseurl }}" class="btn cl-btn github menu-item">Install</a>
+            </nav>
+            <nav class="mobile" role="navigation">
+                <a href="#" title="Expand Menu" class="hamburger">
+                    <div class="hamburger_inner"></div>
+                </a>
+                <a target="_blank" href="{{ "/install-cockroachdb.html" | prepend:site.baseurl }}" class="btn cl-btn github menu-item">Install</a>
+            </nav>
         </div>
-        <div class="navbar-header-items">
-            <ul>
-                <li><a href="{{ "/frequently-asked-questions.html" | prepend:site.baseurl }}">FAQ</a></li>
-                <li><a href="http://www.cockroachlabs.com/docs/">Docs</a></li>
-                <li><a href="http://www.cockroachlabs.com/community/">Community</a></li>
-                <li><a href="http://www.cockroachlabs.com/about/">About</a></li>
-                <li><a href="http://www.cockroachlabs.com/blog/">Blog</a></li>
-                <li><a href="{{ "/install-cockroachdb.html" | prepend:site.baseurl }}" class="btn">Install</a></li>
-            </ul>
-        </div>
-<!--    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```

Comment:
Please leave this commented-out code in place. There's some conditional logic that plugs into some of the theme's conditional logic capabilities, which we don't need now but could come in handy later.